documentation/PATh/support_and_training/support/getting-help-from-RCFs.md

```diff
@@ -0,0 +1,59 @@
+---
+path:
+  path: support_and_training/support/getting-help-from-RCFs.md
+---
+
+Email, Office Hours, and 1-1 Meetings 
+====================================
+ 
+
+*There are multiple ways to get help from PATh’s Research Computing Facilitators. Get in touch anytime!*
+
+
+## Research Computing Facilitators
+
+To help researchers effectively utilize computing resources, our
+Research Computing Facilitators (RCFs) not only assist you in
+implementing your computational work on PATh compute resources, but can
+also point you to other services related to research computing and data
+needs. For example, RCFs can:
+
+- Assist with planning your computational approach for a research problem
+- Teach you to submit jobs via the OSG Connect service
+- Help you with troubleshooting on OSG systems
+- Connect you with other researchers using similar software or methods
+- Point to learning materials for programming and software development
+- Help you identify applicable non-OSG data storage options
+- Find someone who knows the answer to your question, even if the RCF doesn’t
+- … and other helpful activities to facilitate your use of cyberinfrastructure
+
+
+## Help via Email
+
+We provide ongoing support via email to support@osg-htc.org, and it’s
+never a bad idea to start by sending questions or issues via email. You
+can typically expect a first response within a few business hours.
+
+
+## Virtual Office Hours
+
+Drop-in for live help, starting January 11, 2022!
```

Comment:
or earlier? we could start doing this now...

Reply:
Why wait - I think we are ready